pkg/ddc/jindo/create_volume.go

```diff
@@ -30,24 +40,100 @@ func (e *JindoEngine) CreateVolume() (err error) {
 
 // createFusePersistentVolume
 func (e *JindoEngine) createFusePersistentVolume() (err error) {
```

Comment:
suggest adding a pkg named `render` for rendering kube native resource 
convenient for reuse